### DISCUSSION TITLE
Multiimage to gif

### DISCUSSION THREAD

**Discussion by Marshal4582** [YetiForceCompany/YetiForceCRM#15500] [General]
As everybody knows, movie sayz more that still picture. I wonders, what could be done to to convert automatically all images (uploaded to certain multimage field) right to animated gif (can be other single image field)
Does any of you guys tried this?
Imagine some email with information about property. See this:

![unnamed](https://user-images.githubusercontent.com/50770738/143997280-38c032ec-41a1-41a7-bd66-e4335251b873.gif)
:

**Comment by mariuszkrzaczkowski** [comment]
you have to write a custom function to the textparser
and use some function to create a GIF e.g.
https://stackoverflow.com/questions/9417762/make-an-animated-gif-with-phps-imagemagick-api
https://coderedirect.com/questions/2576/make-an-animated-gif-with-phps-imagemagick-api
